plugins/com.google.cloud.tools.eclipse.preferences/src/com/google/cloud/tools/eclipse/preferences/PreferenceInitializer.java

```diff
@@ -11,7 +11,7 @@
   @Override
   public void initializeDefaultPreferences() {
     IPreferenceStore store = Activator.getDefault().getPreferenceStore();
```

Comment:
I wonder if it makes sense to get a preference store from this Activator under the new mini-framework?